aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
+	- Make sure that the **Application Pool Process Model Identity** is either set to **ApplicationPoolIdentity**; or if a custom identity is in use, confirm the identity has permissions to read, modify, and execute on the application's folder.
 
-  - You haven't enabled the proper roles. See `IIS server configuration`_.
+- **Browser:** 502.3 Bad Gateway: There was a connection error while trying to route the request.
+- **Application Log:** Process '0' failed to start. Port = PORT, Error Code = '-2147024894'.
+- **ANCM Log:** Log file created but empty
 
-- HTTP 500.19 : The requested page cannot be accessed because the related configuration data for the page is invalid.
+	- Check the `processPath` attribute on the `\<aspNetCore\>` element in `web.config` to confirm that it is `dotnet` for a portable application or `.\\my_application.exe` for a self-contained application.
+	- You may have deployed a portable application without installing .NET Core on the server. If you are attempting to deploy a portable application and have not installed .NET Core, run the **.NET Core Installer** on the server. See `Install .NET Core on the Server`_.
+	- You may have deployed a portable application and installed .NET Core without restarting the server. Restart the server.
 
-  - You haven't installed the correct HTTP Platform Handler. See `Install the HTTP Platform Handler`_
-  - The *wwwroot* folder doesn't have the correct permissions. See `IIS server configuration`_.
+- **Browser:** 502.3 Bad Gateway: There was a connection error while trying to route the request.
+- **Application Log:** Process 'PROC_ID' failed to start. Port = PORT, Error Code = '-2147023829'.
+- **ANCM Log:** Unhandled Exception: System.FormatException: Unrecognized argument format **--OR --** Expected to load required hostpolicy.dll from [IIS_WEBSITE_PHYSICAL_PATH] - This may be because of an invalid .NET Core FX configuration in the directory.
 
-- The IIS 7.0 CoreWebEngine and W3SVC features must be installed to use the Microsoft HTTP Platform Handler 1.x.
+	- Examine the `arguments` attribute on the `\<aspNetCore\>` element in `web.config` to confirm that it is either (a) `.\\my_applciation.dll` for a portable application; or (b) not present, an empty string (i.e., `arguments=""`), or a list of your application's arguments (i.e., `arguments="arg1, arg2, ..."` for a self-contained application.
 
-  - Enable IIS; see `IIS server configuration`_.
+- **Browser:** 503 Service Unavailable
+- **Application Log:** No entry
+- **ANCM Log:** Log file not created
 
-- HTTP 502.3 Bad Gateway
+	- Confirm that the Application Pool is not in the `Stopped` state.
 
-  - You haven't installed the correct HTTP Platform Handler. See `Install the HTTP Platform Handler`_
+Additional Resources
+^^^^^^^^^^^^^^^^^^^^
 
-- HTTP 500.21 Internal Server Error.
+- `Understanding ASP.NET 5 Web Apps <http://docs.asp.net/en/latest/conceptual-overview/understanding-aspnet5-apps.html>`_
```

Comment:
This was just because the current (RC1) doc is titled this way at that link.

I'll go "Core" and guess that the doc location will be:
```
http://docs.asp.net/en/latest/conceptual-overview/understanding-aspnetcore-apps.html
```